main.py

```diff
@@ -25,93 +25,32 @@ def __init__(self):
 
         # Used when moving two frames
         self.selected_frame = None
+        self.fullscreen = False
 
     def setup_ui(self):
         """Loads the main.ui file and sets up the window and grid."""
-        self.ui = uic.loadUi("ui/main.ui")
+        self.ui = uic.loadUi("ui/main.ui", self)
         self.grid = VideoFrameGrid()
 
         self.container = self.ui.findChild(QtCore.QObject, "container")
         self.container.addLayout(self.grid)
+        self.menubar = self.ui.findChild(QtCore.QObject, "menuBar")
```

Comment:
yeah that's true, but I'll change it anyway. I was just a bit lazy at the time ;)